chevah_build

```diff
@@ -342,10 +347,12 @@ case $OS in
         export CXX="clang++"
         export CFLAGS="$CFLAGS -mmacosx-version-min=10.8"
         export MACOSX_DEPLOYMENT_TARGET=10.8
-        # OS X 10.8 not supported by cryptography 1.9, so we use 1.8.2.
+        # OS X 10.8 not supported by cryptography 1.9, so you'll normally use
+        # 1.8.2, but cryptography 2.2.2 has a working wheel for MacOS 10.6+.
         PIP_LIBRARIES="\
             setproctitle==${SETPROCTITLE_VERSION} \
-            cryptography==1.8.2 \
+            pycryptodome==${PYCRYPTODOME_VERSION} \
+            cryptography==${CRYPTOGRAPHY_VERSION}upstream \
```

Comment:
what is this upstream?

Reply:
As described in the PR description: __[…]For now we use the upstream wheel on these platforms for the current version. The way we do it should make it painfully obvious when upgrading the generic version (defined at the beginning of chevah_build), to avoid these platforms being left behind when incrementing the generic version._

And: _Don't bother building old cryptography 1.8.2 on OS X, we have a working 2.2.2 wheel from upstream. In OS X, OpenSSL 0.9.8 is only used for the standard ssl module now. However, stop using that wheel on MacOS, where we want OpenSSL updates through Homebrew._